frontend/src/components/modal/job/JobDetailModal.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+  <v-row justify="center">
+    <v-dialog v-model="dialog" persistent>
+      <v-card>
+        <v-card-text>
+          <v-row>
+            <v-col cols="" sm="" class="info-col">
+              <v-row>
+                <v-col cols="12" sm="6">
+                  <v-text-field
+                    label="Position*"
+                    class="text-h5"
+                    v-model="jobData.position"
+                    :style="{ color: this.positionErrorIndicator }"
```

Comment:
Should this input be restricted to numeric? I think there's a `number` prop you can pass in.

Reply:
This is like job title / job position not column/status (assuming thats what you were thinking) so should be string input